test/internal/laws/Chain.js

```diff
@@ -0,0 +1,19 @@
+'use strict';
+
+var compose = require('./compose');
+var flatMap = require('./flatMap');
+var forall = require('./forall');
+
+
+module.exports = function(equals) {
+  return forall({
+
+    //  m >>= f >>= g = m >>= (f >=> g)
```

Comment:
As I mentioned in my previous comment (in another thread) I'd like to stick to a two-function version even though the three-function `>=>` version is so elegant. :fish:

> I think we should add parentheses to (m >>= f)

Parens appear not to be required:

```haskell
λ Just 10 >>= Just . (* 2) >>= Just . (+ 1)
Just 21
```

Are you suggesting we add parens for clarity?